src/main/docs/guide/metricsAndReporters/metricsAndReportersNewRelicTelemetry.adoc

```diff
@@ -0,0 +1,32 @@
+A New Relic Registry using https://github.com/newrelic/micrometer-registry-newrelic[New Relic's own Micrometer Registry] rather than the New Relic registry provided by the Micrometer project.
+
+You can include the New Relic Telemetry reporter via `io.micronaut.configuration:micronaut-micrometer-registry-new-relic-telemetry:${micronaut.version}`
```

Comment:
Ideally, the version should be removed from here and added to a project BOM (#85)